src/hexen/p_acs.c

```diff
@@ -787,6 +789,23 @@ static int GetACSIndex(int number)
 
 //==========================================================================
 //
+// CheckACSPresent
+//
+// Placing Korax in a PWAD without extra steps will result in a crash in
+// Vanilla because the relevant ACS scripts are not initialised
+//
+//==========================================================================
+
+void CheckACSPresent(int number)
+{
+    if (GetACSIndex(number) == -1)
```

Comment:
I'm not sure whether to just so this test a second time in     P_StartACS and eliminate the need for a new subroutine. At the moment, I can guarantee that this new behaviour only takes effect for the Korax scripts; modifying     P_StartACS would mean a more thorough audit for the implications.